src/fragments/forms/map-form/components/isochrones/i18n/isochrones.i18n.cs-cz.js

```diff
@@ -0,0 +1,18 @@
+
+export default {
+  isochrones: {
+    isochrones: 'Isochrony',
+    isochronesReady: 'Isochrony připraveny',
+    calculatingIsochrones: 'Výpočet isochron ...',
```

Comment:
The translation for "isochrone" is written with an `s` here and with a `z` in lines 7, 14 and 16.
Is both correct in czech?